oauth_sample/config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    CLIENT_ID: '',
+    CLIENT_SECRET: '',
```

Comment:
Suggestion: you could set a good example here of not checking secrets into git by adding `config.js` to `.gitignore`, renaming this to `config.js.example`, and adding another setup step to the readme: `cp config.js.example config.js`. That way people can fill in their client ID/secret but they won't show up in the git working tree afterwards:

```
$ git status
On branch af/publish-oauth-sample-application-qppauth-852
Your branch is up to date with 'origin/af/publish-oauth-sample-application-qppauth-852'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   config.js
```

Reply:
Yeah that is a good callout, I was actually considering something like that because I always double check that `CLIENT_ID` and `CLIENT_SECRET` are empty before committing!

Will do that and update the `README.md` as well.